docs/inspector.md

```diff
@@ -0,0 +1,43 @@
+# Hermes Inspector
+
+React Native for Windows with Hermes engine now supports direct JavaScript runtime inspection using tools such as Chrome or Edge devtools, VSCode debugger, Flipper etc. by implementing an in-process Chrome Debug Protocol server.  Please note that it is fundamentally different from "Remote JS Debugging", which loads the JavaScript bundle into a remote Chrome browser session with dupelx communication over IPC channels.
+
+We share the implementation (code and design) with other platforms wherever possible. All the external endpoints, APIs and protocols should be identical to React-Native environments on other platforms. Hence, we expect most tooling available on other platforms to just work on Windows. But, as of now, we have tested only with Chrome and Edge devtools.
+
+## Steps to enable direct debugging
+
+1. Initialize React Native Host,
+   - Turn on `DeveloperSupport` 
+   - Turn on `FastRefresh`
+   - Turn off `WebDebugger`
+   - Turn on `Direct Debugging`
+2. Ensure Dev-Server is running
+3. Start the application
+
+After the app has booted,
+
+4. Navigate to `edge:\\inspect` in Edge browser or `chrome:\\inspect` in Chrome browser
```

Comment:
>`edge:\\inspect [](http://example.com/codeflow?start=15&length=15)

It must be forward slashes for the protocols: edge://inspect. The same is for the chrome protocol.